samples/net/sockets/echo/README.rst

```diff
@@ -76,3 +76,22 @@ To test:
 
 As can be seen, the behavior of the application is the same as the Zephyr
 version.
+
+Running on cc3220sf_launchxl
+============================
+
+In this configuration, WiFi, and the networking wifi shell are enabled.
+This allows the user to connect to an access point to obtain
+an IP address.
+
+To connect to an AP:
+
+.. code-block:: console
+
+    shell> select wifi
+    wifi> scan
+    wifi> connect "<mySSID>" <SSID_len> 0 <myPassword>
+
+replacing <*> tokens with the actual SSID name, length, and password.
```

Comment:
change to

     ``<*>``

Reply:
Thanks.
Actually, when the patch is re-based and "fast connect" feature enabled, these instructions get simplified.  The mention to ''<*>" is removed.  Next push will have an updated section on cc3220sf_launchxl.